### DISCUSSION TITLE
I did a small podcast about TW recently

### DISCUSSION THREAD

**Discussion by kwoot** [GothenburgBitFactory/taskwarrior#2481] [General]
Hi,

I did a small podcast about TW some months ago.

I got some feedback from the podcast and I like to share these here so the devs can read them.
http://hackerpublicradio.org/eps.php?id=3226

Thanks for a tool I use daily now.

**Comment by tbabej** [comment]
Nice episode. The "setting" for forgetting the tasks, that you couldn't recall on the spot is the `until` attribute: https://taskwarrior.org/docs/using_dates.html

**Reply by kwoot** [reply]
Glad you liked it ! And yes, I found it (until) later and it is definitely now part of my routine.
